src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -48,8 +48,9 @@ public async Task<IActionResult> GetBestEmissionsDataForLocationsByTime([FromQue
                 { CarbonAwareConstants.Best, true }
             };
 
-            var response = await _aggregator.GetBestEmissionsDataAsync(props);
-            return response != null ? Ok(response) : NoContent();
+            IEnumerable<EmissionsData> response = await _aggregator.GetBestEmissionsDataAsync(props);
```

Comment:
Need to also change [line 33](https://github.com/Green-Software-Foundation/carbon-aware-sdk/pull/104/files#diff-1f2e99bbca80e13b4196ed490313cd965b101c63cd15413f404da07b0a1e97deR33) to  `[ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<EmissionsData>))]` so the API docs reflect this update.

This would be a breaking change.  I'm not sure if there is a standard way of signaling that to existing API users.  `CHANGE_LOG.md` ?  Release notes?  Nothing is in `main` yet, so there are no breaking changes?

Reply:
Yeah, my view is it's breaking for downstream current users but with no 1.0 release it's not a breaking change as such (technically) ... I think a CHANGE_LOG.md might be useful in dev still.